pom.xml

```
@@ -19,7 +19,7 @@
   <properties>
     <revision>6.0</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jetty.version>9.4.48.v20220622</jetty.version>
+    <jetty.version>10.0.9</jetty.version>
```

This version is a few versions behind. I suggest using the latest version:

```suggestion
    <jetty.version>10.0.11</jetty.version>
```

---

already done. maybe just wait I mark it ready for review.